testlist.sh

```diff
@@ -1,20 +1,172 @@
 #!/bin/sh
 
+# This is the wrapper script to launch scale tests.
+
+declare -a test_array
+declare -A IOR_suite
+declare -A MDT_suite
+
+# Run all three test suites when no argument is given, else the specific test(s) given as arguments
+if [ $# -eq 0 ] ; then
+    test_array=(IOR MDT SELF_TEST)
+else 
+    test_array=("$@")  
+fi
+echo Tests to run are ${test_array[*]}
+
+# IOR test suite
+# Format:[<test name>]="min_servers max_servers num_targets num_ranks replication_type stonewall_timeout run/no_run"
+IOR_suite=( [16Clnt_IOReasy_SX]="1 256 16 32 SX 300 YES" [16Clnt_IORhard_SX]="1 256 16 32 SX 300 YES"
```

Comment:
It is very hard to tell what you are running in the testcase here without going through the rest of the code in this script. For example, one cannot tell how you are scaling the test by the arguments used here.
There is also no way to figure from here what each arg means in the hash key string without studying the code below.